app/src/main/java/org/merakilearn/ui/playground/PlaygroundFragment.kt

```diff
@@ -70,8 +67,17 @@ class PlaygroundFragment : BaseFragment() {
                     requireActivity(),
                     file = it.file
                 )
+                is PlaygroundViewEvents.OpenScratch -> {
+                    val intent = Intent(requireContext(), ScratchActivity::class.java)
+                    startActivity(intent)
+                }
+                is PlaygroundViewEvents.OpenScratchWithFile -> {
+                    val intent = Intent(requireContext(), ScratchActivity::class.java)
+                    intent.putExtra("file", it.file)
```

Comment:
better if we create a variable in some utils/constants class and use 'file' intent extra keyword from there, else it becomes hard to keep track and make changes.